.github/workflows/ecosystem-recompile.yml

```
@@ -0,0 +1,43 @@
+name: Ecosystem | Recompile members.json
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+
+jobs:
+    runs-on: ubuntu-latest
+    env:
+      tox_env: "py39"
+      python-version: "3.9"
+    steps:
```

To avoid concurrency runs you can add something like this before the `steps` :
```suggestion
    concurrency: ci-ecosystem-recompile
    steps:
```

---

Nice, thanks

---

Added in [dd3e11a](https://github.com/qiskit-community/ecosystem/pull/426/commits/dd3e11a9db18e53777ad05b4fb98664e8d8cfe72)